dotcom-rendering/src/web/components/HeaderSingleFrontDoor.tsx

```diff
@@ -33,7 +33,7 @@ export const HeaderSingleFrontDoor = ({
 	idApiUrl,
 }: Props) => (
 	<div css={headerStyles}>
-		<Island deferUntil="idle">
+		<Island>
```

Comment:
Why not wait? Is the priority of this increased? This may be a candidate for `expediteIsland`.

Reply:
Ah nice, yeah I wanted to start a discussion about this. I removed the deferUntil hoping that the client side rendering would kick in sooner so that the switch between 'sign in' and 'my account' in the menu happens as quickly as possible. But I wanted to check with you guys if you thought this was the right thing to do

Reply:
Yes, it should be relatively safe to change it for now. We can look at improving further later.